core/src/test/kotlin/org/eln2/sim/electrical/mna/CircuitTest.kt

```diff
@@ -19,11 +19,12 @@ internal class CircuitTest {
 
         init {
             c.add(vs, r1)
-            vs.neg.named("vneg")
-            vs.pos.named("vpos")
+            // TODO: figure out how to name nodes early again
+            // vs.neg.named("vneg")
+            // vs.pos.named("vpos")
             vs.connect(1, r1, 1)
             vs.connect(0, r1, 0)
-            vs.connect(1, c.ground)
+            vs.ground(1)
```

Comment:
+1 I like this syntax better.

Reply:
It really was the only special case; that should have happened forever ago :P